docs/source/api.rst

```diff
@@ -199,3 +199,6 @@ Enums
 
 .. autoclass:: voltage.RelationshipType
 
+Event Listeners
+===============
+.. attributetable:: voltage.Client.listen
```

Comment:
is listen a class? and its just the method so it wont automatically list every event

Reply:
it's a method that's already documented because client is automatically documented

Reply:
what you're *supposed* to do is add a new section for listeners right under the client section and provide a list of the listener's names, their corresponding api events, maybe a simple explanation and the arguments they return